web/themes/custom/server_theme/src/js/toggler.js

```diff
@@ -14,8 +14,8 @@
    * @type {{attach: Drupal.behaviors.serverThemeToggler.attach}}
    */
   Drupal.behaviors.serverThemeToggler = {
-    attach: function(context, settings) {
-      const $buttons = $(once('server-theme-button', 'button[aria-controls]', context)).not('button[aria-controls="admin-toolbar"]');
+    attach: function (context, settings) {
+      const $buttons = $(once('server-theme-button', 'button[aria-controls], button:not(.toolbar-button)', context)).not('button[aria-controls="admin-toolbar"]');
```

Comment:
Untested, but maybe we can avoid having two `not`

```suggestion
        const $buttons = $(once('server-theme-button', 'button[aria-controls]:not([aria-controls="admin-toolbar"]), button:not(.toolbar-button)', context));
```

Reply:
@amitaibu Indeed, that's a good Idea. Thanks for the suggestion. Updated the PR, Please check it.

#### Buttons before code change:
![image](https://github.com/user-attachments/assets/b4b595b6-ea66-4f8e-9e6d-b47d08ec5e7d)

#### Buttons after code change:
![image](https://github.com/user-attachments/assets/fa2b9dcd-6ba9-4952-b4b7-5d734eca29ee)

Reply:
I think the code is still calling the `not` method `$(...).not(...)`.

Can it be changed to `const $buttons = $(once('server-theme-button', 'button[aria-controls]:not([aria-controls="admin-toolbar"]), button:not(.toolbar-button)', context));`?

Reply:
@amitaibu Did you get a chance to look at this?

Reply:
Yes, I still think we can avoid the `.not()` function, and have it inside the once :)